quickannotator/db/logging.py

```diff
@@ -0,0 +1,27 @@
+from quickannotator.db import get_session
+import logging
+import quickannotator.db.models as models
+from datetime import datetime
+
+class SQLAlchemyHandler(logging.Handler):
+    def emit(self, record):
+        with get_session() as db_session:
+            log_entry = models.Log(
+                name=record.name,
+                timestamp=datetime.fromtimestamp(record.created),
+                level=record.levelname,
+                message=self.format(record)
+            )
+
+            db_session.add(log_entry)
+
+
+def init_logger(logger_name) -> logging.Logger:
+    logger = logging.getLogger(logger_name)
+    logger.setLevel(logging.DEBUG)
```

Comment:
for sanity sake, i think default level in most places is info?

Reply:
Sure, pushed this in on Friday.

I think this PR is ready to be merged unless you have any other requests?

I can also add documentation for setting up the grafana instance? The setup is very easy and only needs to be done once since it runs in a separate docker container.

Reply:
okay for merge - i think everything was looking good

> I can also add documentation for setting up the grafana instance? The setup is very easy and only needs to be done once since it runs in a separate docker container

yes, probably will need this for the other developers (me, nan, etc)